### DISCUSSION TITLE
Can't run via snap

### DISCUSSION THREAD

**Discussion by dirtydurkin101** [sct/overseerr#832] [Help] [answered]
I just get the error 'cannot find app 'overseerr in overseerr'. 
I have to use it this way as I'm running via raspberry pi and docker doesn't allow me to create a container due to incorrect architecture.

**Comment by samwiseg0** [comment]
@dirtydurkin101 Can you stop by discord to troubleshoot? The snap packages should work as others have them working. I would like more information on the error. Feel free to ping me when you get into discord. I have the same handle on there too.

**Comment by samwiseg0** [comment, marked as answer]
Resolved.
